picard/tagger.py

```diff
@@ -234,6 +245,18 @@ def __init__(self, picard_args, localedir, autoupdate, pipe_handler=None):
         # Default thread pool
         self.thread_pool = ThreadPoolExecutor()
 
+        self.commands = {
+            "CLUSTER": self.handle_command_cluster,
+            "FINGERPRINT": self.handle_command_fingerprint,
+            "LOOKUP": self.handle_command_lookup,
+            "QUIT": self.handle_command_quit,
+            "REMOVE_SAVED": self.handle_command_remove_saved,
+            "SAVE_COMPLETE": self.handle_command_save_complete,
+            "SCAN": self.handle_command_scan,
+            "SHOW": self.handle_command_show,
+            # SUBMIT_FINGERPRINTS ???
```

Comment:
I cannot seem to find a function submitting fingerprints, any ideas?

Reply:
> I cannot seem to find a function submitting fingerprints, any ideas?

https://github.com/metabrainz/picard/blob/695d2aa7597fb6723da06ca6081413ffe5fcd58a/picard/acoustid/manager.py#L153